windows_install_dependencies.bat

```diff
@@ -145,12 +158,133 @@ set "bazel_path=%opt_install_dir%\bazel.exe"
 IF /I EXIST %bazel_path% (
     if %expunge% EQU 1 (
         rm -rf %bazel_path%
+        if %errorlevel% neq 0 exit /b %errorlevel%
         wget -O %bazel_path% https://github.com/bazelbuild/bazel/releases/download/6.4.0/bazel-6.4.0-windows-x86_64.exe
+        if %errorlevel% neq 0 exit /b %errorlevel%
     ) else (
         echo [INFO] ::::::::::::::::::::::: bazel already installed
     )
 ) ELSE (
     wget -O %bazel_path% https://github.com/bazelbuild/bazel/releases/download/6.4.0/bazel-6.4.0-windows-x86_64.exe
+    if %errorlevel% neq 0 exit /b %errorlevel%
+)
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::::::::::::::::::::::: Python39
+set "python39_path=%opt_install_dir%\Python39\"
+set "python39_system=C:\Program Files\Python39\"
```

Comment:
Why do we consider both options? I suppose Python 3.9 is not a default option for Windows now.

Reply:
C:\Program Files\Python39\ this is preinstalled on CI nodes. We use it for the purpose of setting python39 for ovms builds. If the user also has this path or has already installed python in c:/opt the install dependencies will work on laptop. Otherwise it will fail and tell user to install Python.